### DISCUSSION TITLE
How are unnamed faces sorted?

### DISCUSSION THREAD

**Discussion by jagjordi** [immich-app/immich#3327] [Q&A]
Is it by number of occurances?

**Comment by bo0tzz** [comment]
Yes. The sort happens here: https://github.com/immich-app/immich/blob/main/server/src/infra/repositories/person.repository.ts#L58

**Reply by jagjordi** [reply]
Thank you!

**Reply by jrasm91** [reply]
And named people always come before people without names (named people are also ordered by face count)
